minidns-core/src/main/java/de/measite/minidns/DNSClient.java

```diff
@@ -154,4 +125,20 @@ public static synchronized void addDnsServerLookupMechanism(DNSServerLookupMecha
     public static synchronized boolean removeDNSServerLookupMechanism(DNSServerLookupMechanism dnsServerLookup) {
         return LOOKUP_MECHANISMS.remove(dnsServerLookup);
     }
+
+    public boolean isAskForDnssec() {
+        return askForDnssec;
+    }
+
+    public void setAskForDnssec(boolean askForDnssec) {
+        this.askForDnssec = askForDnssec;
+    }
+
+    public boolean isDisableResultFilter() {
+        return disableResultFilter;
+    }
+
+    public void setDisableResultFilter(boolean disableResultFilter) {
+        this.disableResultFilter = disableResultFilter;
+    }
```

Comment:
dnssecEnabled would be misleading as no dnssec validation is done, it just sets the "dnssec OK" bit in the request to receive RRSIG records. This is the same as +dnssec flag in dig, and for dig I thing the flag is confusing as well.

regarding result filtering: yes